.github/workflows/publish.yml

```diff
@@ -15,9 +15,11 @@ jobs:
     needs: validate
     runs-on: ubuntu-latest
     permissions:
+      actions: write
```

Comment:
This is needed to cancel the GitHub Action run (in case there are no publishing to actually do)